bang/forms.py

```diff
@@ -424,12 +424,18 @@ class EventFilterForm(MagiFiltersForm):
         ('japanese_name', string_concat(_('Title'), ' (', t['Japanese'], ')')),
     ]
 
+    def boost_members_to_queryset(self, queryset, request, value):
+        return queryset.filter(boost_members=value)
+
+    boost_members = forms.ChoiceField(choices=BLANK_CHOICE_DASH + [(id, full_name) for (id, full_name, image) in getattr(django_settings, 'FAVORITE_CHARACTERS', [])], initial=None, label=_('Boost Member'))
+    boost_members_filter = MagiFilter(to_queryset=boost_members_to_queryset)
```

Comment:
Default behavior does that because `boost_members` is already part of event model so it knows how to filter. This line + boost_members_to_queryset can be removed!

Reply:
✅